code/modules/mob/living/simple_animal/abnormality/he/wayward_passenger.dm

```diff
@@ -0,0 +1,231 @@
+/mob/living/simple_animal/hostile/abnormality/wayward
+	name = "Wayward Passenger"
+	desc = "A large humanoid with its torso caved open and lined with teeth. Thread-like projections cover its open wounds."
+	icon = 'ModularTegustation/Teguicons/48x96.dmi'
+	icon_state = "wayward"
+	icon_living = "wayward_breach"
+	icon_dead = "wayward_dead"
+	del_on_death = FALSE
+	maxHealth = 1200
+	health = 1200
+
+	move_to_delay = 4
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.7, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 0.7, PALE_DAMAGE = 1.5)//lovetown residents LOVE physical pain, so highly resistant to black and red
+	stat_attack = HARD_CRIT
+
+	melee_damage_lower = 20
+	melee_damage_upper = 24
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_sound = 'sound/abnormalities/wayward_passenger/attack2.ogg'
+	can_breach = TRUE
+	can_buckle = FALSE
+	vision_range = 14
+	aggro_vision_range = 20
+	threat_level = HE_LEVEL
+	start_qliphoth = 1
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(50, 55, 55, 55, 60),
+						ABNORMALITY_WORK_INSIGHT = list(40, 30, 20, 40, 40),
+						ABNORMALITY_WORK_ATTACHMENT = 30,
+						ABNORMALITY_WORK_REPRESSION = list(55, 60, 60, 60, 55)
+						)
+	work_damage_amount = 11
+	work_damage_type = RED_DAMAGE
+	fear_level = WAW_LEVEL
+	max_boxes = 16
+	base_pixel_x = -8
+	pixel_x = -8
+
+	ego_list = list(
+		/datum/ego_datum/weapon/warp,
+		/datum/ego_datum/weapon/warp/spear,
+		/datum/ego_datum/armor/warp
+		)
+//	gift_type =  /datum/ego_gifts/warp
+	gift_message = "This lighter is branded with a certain company logo."
+	abnormality_origin = ABNORMALITY_ORIGIN_LIMBUS
+
+	//teleport vars
+	var/teleport_cooldown
+	var/teleport_cooldown_time = 10 SECONDS
+	var/can_act = TRUE
+	//dash vars
+	var/charging = FALSE
+	var/dash_num = 6
+	var/dash_cooldown = 0
+	var/dash_cooldown_time = 4 SECONDS
+	var/list/been_hit = list() // Don't get hit twice.
+
+//*** Simple mob procs ***
+/mob/living/simple_animal/hostile/abnormality/wayward/death()
+	density = FALSE
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/wayward/Life()
+	. = ..()
+	if(IsContained())
+		return
+	if(client)
+		return
+	if(teleport_cooldown <= world.time)
+		TryTeleport()
+	return
+
+/mob/living/simple_animal/hostile/abnormality/wayward/Move()
+	if(!can_act)
+		return FALSE
+	return ..()
+
+//*** Work mechanics ***
+/mob/living/simple_animal/hostile/abnormality/wayward/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(65))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/wayward/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(10))
+		datum_reference.qliphoth_change(-1)
+	return
```

Comment:
These are fairly different %s when compared to other abnormalities that share similar wording. To my knowledge, "Low/Normal/High" probability is "20/40/80" respectively. Not saying you have to change them to be those exact numbers but I do think that the 10% is a bit low and 20% may be better.

Reply:
changed it to 75% and 20%, respectively